lint/highlight.py

```diff
@@ -22,12 +22,19 @@
 
 
 class RegionStore:
-    def __init__(self):
+    _memory = None
+
+    @property
+    def memory(self):
         """structure: {"view.id": [region_keys ... ]}."""
-        self.memory = sublime.load_settings("sl_regions.sublime-settings")
-        views = self.memory.get("views")
-        if not views:
-            self.memory.set("views", {})
+        memory = self._memory
+        if not memory:
+            memory = sublime.load_settings("sl_regions.sublime-settings")
```

Comment:
Don'

Reply:
We never call `save_settings` if that matters.

Reply:
first question: I don't know

second: we only ever *read* settings, so we shouldn't have a need for `save_settings `

Reply:
But it looks like someone wanted to persist these regions across sessions. And we could use `save_settings` to do that. But it is not implemented. @pykong did this.

If we just use a normal *dict*, we don't need a lazy *property* here.

Reply:
Right, that sounds like a remnant of our earlier iterations. I think SL3 had a lot of caching and persistence, etc., that we don't need anymore. IMO you're free to change this.

Reply:
@kaste @braver we could change that indeed if everything works as before.

The idea behind the in-memory settings was to persist region throughout a session.
So, when there is an exception that SublimeLinter will be given the chance to pick up those then remnant highlights from the previous run and clear them.